RELEASE_NOTES.md

```diff
@@ -1,5 +1,118 @@
+file[/tmp/foo.xzy] 0.015114     test     default test/recipes/default.rb:2:in `from_file'
+
+  - Chef::Handler::SlowReport
+Running handlers complete
+Chef Infra Client finished, 0/1 resources updated in 03 seconds
+```
+
+Add a slow resources report to chef-client - https://github.com/chef/chef/pull/11642
+
+### Improved YAML Recipe Support
+
+Chef Infra Client now supports YAML recipes that end in `.yaml` in addition to those ending in `.yml`. If a `.yml` and `.yaml` recipe of the same name is present Chef Infra Client will now fail as there is no way to determine which recipe should be loaded in this case.
+
+#### Improved Reporting to Automate
+
+Chef Infra Client run reporting to Automate now respects attribute `allowlist` and `denylist` configurations set in the `client.rb` configuration. This change allows users to limit the data sent to their Automate servers to prevent indexing sensitive data, or to reduce the necessary storage space on the Automate server.
+
+### Updated Resources
+
+#### homebrew_path
+
+The `homebrew_path` now passed the `homebrew_path` when creating or deleting taps. This change prevents failures when running homebrew in a non-standard location or on a M1 system. Thanks [@mattlqx](https://github.com/mattlqx)!
+
+#### hostname
+
+The `hostname` resource now sets the hostname on Windows systems using native PowerShell calls for increased reliability and allows changing the hostname on domain-attached systems. To change the hostname on a domain-attached system pass a domain administrator account using the new `domain_user` and `domain_password` properties.
+
+#### openssl_x509_certificate
+
+The `openssl_x509_certificate` no longer marks the creation of the X509 certificate file as sensitive since this makes troubleshooting difficult and this content is not sensitive. Thanks [@jasonwbarnett](https://github.com/jasonwbarnett)!
+
+#### windows_firewall_rule
+
+The `windows_firewall_rule` resource now allows specifying multiple IP addresses in the `remote_address` property.
+
+#### windows_pagefile
+
+The `windows_pagefile` resource features improved performance and support for the latest releases of Windows 10. The behavior of the resource has also been improved to make managing pagefiles more predictable:
+
+The `path` property now accepts a drive letter where the pagefile should be stored in addition to the full path the pagefile on disk. For example `C`, `C:`, or `C:\` can now be used to specify a pagefile stored at `C:\pagefile.sys`.
+
+Creating a new pagefile will no longer disable the system managed pagefile by default. If you wish to create a pagefile while also disabled the system managed pagefile set `system_managed false`.
+
+#### windows_printer
+
+The `windows_printer` resource includes improved logging when adding or removing printers.
+
+#### windows_printer_port
+
+The `windows_printer_port` resource has been refactored with several improvements:
```

Comment:
Some of these sentences would read "improvements improved", so I changed the wording.